aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -110,6 +130,12 @@ module aptos_framework::account {
         });
     }
 
+    public(friend) fun create_address_map(aptos_framework_account: &signer) {
```

Comment:
Can you add a comment that this will only be called during genesis? Also adds a check that aptos_framework_account is actually the aptos framework account